ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/fieldTemplates/longDescription.html

```diff
@@ -1,6 +1,6 @@
 <template data-sly-template.longDescription="${@ componentId, longDescription, bemBlock}">
   <div aria-live="polite">
-    <div id="${componentId}-longDescription" class="${bemBlock}__longdescription" data-sly-test="${longDescription}">
+    <div id="${componentId}-longDescription" class="${bemBlock}__longdescription ${bemBlock}__longdescription--hidden" data-sly-test="${longDescription}">
```

Comment:
what is the need for adding this extra class ? Any behaviour should be done based on data-attribute

Reply:
Done